client/k8s.go

```diff
@@ -235,9 +235,65 @@ func (k K8sClient) GetcStorPVCs(node string) map[string]*util.Volume {
 			PVC:                     *i.Spec.Source.PersistentVolumeName,
 			CSIVolumeAttachmentName: i.Name,
 			AttachementStatus:       util.CheckVolAttachmentError(i.Status),
-			AccessMode:              util.CheckIfAccessable(i),
+			AccessMode:              k.GetPV(*i.Spec.Source.PersistentVolumeName).Spec.AccessModes,
 		}
 		volumes[vol.PVC] = vol
 	}
 	return volumes
 }
+
+// GetPV used to get pvc information
+func (k K8sClient) GetPV(name string) *corev1.PersistentVolume {
+
+	vol, err := k.cs.CoreV1().PersistentVolumes().Get(name, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("Error while getting volume %s", name)
+	}
+
+	return vol
+}
+
+// GetCVC used to get cStor Volume Config information for cStor a given volume
+func (k K8sClient) GetCVC(name string) *cstorv1.CStorVolumeConfig {
+
+	cStorVolumeConfig, err := k.cStorCS.CStorVolumeConfigs(k.ns).Get(name, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("Error while getting cStor Volume Config for  %s in %s", name, k.ns)
+	}
+
+	return cStorVolumeConfig
+}
+
+// GetCVR used to get cStor Volume Replicas for a given cStor volumes
+func (k K8sClient) GetCVR(name string) []cstorv1.CStorVolumeReplica {
+
+	label := "cstorvolume.openebs.io/name" + "=" + name
+
+	CStorVolumeReplicas, err := k.cStorCS.CStorVolumeReplicas("").List(metav1.ListOptions{LabelSelector: label})
+	if err != nil {
+		klog.Errorf("Error while getting cStor Volume Replica for  %s in %s", name, k.ns)
+	}
+
+	if len(CStorVolumeReplicas.Items) == 0 {
+		klog.Errorf("Error while getting cStor Volume Replica for  %s , couldnot fild any replicas", name)
+	}
+
+	return CStorVolumeReplicas.Items
+}
+
+// NodeForVolume used to get NodeName for the volume
+func (k K8sClient) NodeForVolume(volName string) string {
+
+	label := "openebs.io/persistent-volume" + "=" + volName
+
+	PodInfo, err := k.cs.CoreV1().Pods("").List(metav1.ListOptions{LabelSelector: label})
```

Comment:
Regarding `PodInfo`

We follow small came case unless we want to export. ( e.g. We want to export some fields in go Struct -- we can follow Capital Came Case there )

We can follow Lower Camer Case here. 
For more info : 
https://wiki.c2.com/?LowerCamelCase